### DISCUSSION TITLE
How to enable cors on Nuxt3 server api endpoint

### DISCUSSION THREAD

**Discussion by samiamfong** [nuxt/nuxt#17483] [General]
I am new to nuxt.  Writing my first app with nuxt 3.  I am using the server/api directory as directed to create this endpoint.  Works fine, but I want this endpoint to only service the same domain.  I guess cors is not enabled by default?  Any help is appreciated.

**Comment by wobsoriano** [comment]
I haven't tried but try the [express cors middleware](https://github.com/expressjs/cors). 

```ts
// ~/server/middleware/cors.ts
import cors from 'cors'

export default cors({ origin: 'http://example.com' })
```
